app/services/trends_service.py

```
@@ -0,0 +1,58 @@
+            MonthlyTrendDataPoint(
+                year=row.year,
+                month=row.month,
+                total_spent=row.total_spent
+            )
+            for row in monthly_totals
+        ]
+
+        logger.info(f"Successfully generated monthly trend data for user {user_id}.")
+        return trend_data
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)